File: zerog_scotlandyard/MrX.java
package zerog_scotlandyard;
import static zerog_scotlandyard.TransportType.*;

public class MrX extends Player {
	public MrX(String name, int id){
		super(name, id);
		tickets.put(bus, 4);
		tickets.put(taxi, 3);
		tickets.put(underground, 3);
		tickets.put(blackCard, 2);
	}
}

File: unity/Assets/MrX.cs
using UnityEngine;
using System.Collections;
using System;

public class MrX : Player {
	public MrX(String name, int id) : base(name,id){
		_tickets.Add(TransportType.taxi, 4);
		_tickets.Add(TransportType.bus, 3);
		_tickets.Add(TransportType.underground, 3);
		_tickets.Add(TransportType.blackCard, 2);
	}
}